lib/cndb/cndb.c

```diff
@@ -858,9 +869,16 @@ cndb_compact(struct cndb *cndb)
     map_iter_init(&txiter, cndb->tx_map);
 
     while (map_iter_next(&txiter, &txid, (uintptr_t *)&tx)) {
+        bool can_rollforward = cndb_txn_can_rollforward(tx);
```

Comment:
const

Reply:
Yeah, it's used i just one place, I'll just call the function at that spot.